app/Http/Controllers/WithdrawalController.php

```diff
@@ -385,7 +385,9 @@ public static function close(Request $request, $id)
         }
 
         foreach ($withdrawal->users() as $user) {
```

Comment:
This foreach loop can be removed as we are not editing the bank object anymore after removing the is_first boolean.

Reply:
Yeah you're right, fixed